packages/offix-client/src/config/ApolloOfflineClientConfig.ts

```diff
@@ -17,11 +17,29 @@ import { CacheUpdates } from "offix-cache";
 import { ApolloOfflineQueueListener, createDefaultLink, CompositeConflictListener } from "../apollo";
 import { CachePersistor } from "apollo-cache-persist";
 
+
+interface ApolloOfflineClientConfigIntf extends ApolloOfflineClientOptions {
+  httpUrl?: string;
+  offlineQueueListener?: ApolloOfflineQueueListener;
+  conflictStrategy: ConflictResolutionStrategy;
+  conflictProvider: ObjectState;
+  networkStatus?: NetworkStatus;
+  terminatingLink: ApolloLink | undefined;
+  cacheStorage: PersistentStore;
+  offlineStorage: PersistentStore;
+  conflictListener: CompositeConflictListener;
+  mutationCacheUpdates?: CacheUpdates;
+  cachePersistor?: CachePersistor;
```

Comment:
I think this is what is breaking the build,
it should be:
```
...
cacheStorage: PersistentStore<PersistedData>,
offlineStorage: PersistentStore<PersistedData>,
...
cachePersistor?: CachePersistor<object>
```

However, at that point, it may just be better to use the original `ApolloOfflineClientOptions` interface as this interface would then be identical.

Reply:
I see that ts-interface-builder doesn't support generics though, so I see now why there was a need to create a new interface.

Reply:
Exactly! `ApolloOfflineClientConfigIntf` also fails because of its fields that are generic types